meilisearch-schema/src/schema.rs

```diff
@@ -159,18 +161,18 @@ impl Schema {
         }
     }
 
-    pub fn indexed(&self) -> Cow<[FieldId]> {
+    pub fn indexed(&self) -> Vec<FieldId> {
         match self.indexed {
-            OptionAll::Some(ref v) => Cow::Borrowed(v),
+            OptionAll::Some(ref v) => v.iter().cloned().collect(),
             OptionAll::All => {
                 let fields = self
-                    .fields_map
+                    .indexed_map
```

Comment:
Why have you changed that? Was that a bug in the previous version?

Reply:
this is strictly equivalent in this context, but it seemed easier to understand this way